src/js/components/FileInput/FileInput.js

```diff
@@ -306,6 +352,7 @@ const FileInput = forwardRef(
                   >
                     {theme.fileInput.button ? (
                       <Button
+                        tabIndex={-1}
```

Comment:
Can we add a comment to the places we set the `tabIndex={-1}`?
```suggestion
                        // Button is redundant for keyboard users
                        tabIndex={-1}
```

Reply:
@jcfilben, we sure can. I've amended my commit with the extra commentary.